tokens/duration.yml

```diff
@@ -0,0 +1,16 @@
+props:
+  - name: duration-none
+    value: 0
+  - name: duration-fast
+    value: 100
+  - name: duration-base
+    value: 200
+  - name: duration-slow
+    value: 300
+  - name: duration-slower
+    value: 400
+  - name: duration-slowest
+    value: 500
```

Comment:
Is there a reason those values are unitless?

Reply:
Thanks for ping!

- Whatever solves your problems now is great
- My main focus will be guidelines of usage, rather than spec of tokens
- 50ms increments will be very useful
- Current naming convention looks perfect to me.
- More elaborate token naming is hinged on more elaborate token concepts. We currently have no such concepts or time allocated to create such concepts. An exploration of this would quite naturally follow when we have a couple of UX patterns that we'd want to streamline.

Can't respond to the `units` question, I assume that it's not meant for me.

Reply:
@kaelig right, so - in JS land we want a unitless value, since JS understands that it is milliseconds. In SCSS land we want this as a string with `ms`.

What is the correct solution for this? Do we:
1. use string interpolation when consumed in SCSS to append the `ms`, or
2. write a custom task to handle `duration`, sending the unchanged `number` to everything expect SCSS, where we need to append `ms` and return a `string

Reply:
Thanks @johanstromqvist! 

@beefchimi I looked into it and I think the best way to think about it is to have units in the source tokens (more explicit, could even be authored as seconds or hours), and strip them when appropriate (also: convert them to milliseconds, because in JS that's most useful).

I've opened a PR that should solve this: #55